api/src/typedefs/DashboardTypeDefs.ts

```diff
@@ -0,0 +1,12 @@
+import { gql }  from 'apollo-server'
+
+const DashboardTypeDefs = gql`
+    extend type Query {
+        OngoingEvent : Int!
+        CommingEvent : Int!
+        PastEvent : Int!
+        CurrentUser : Int!
```

Comment:
Hey Dhaval, I think it would be easy for you if you can fetch all data in one query for the dashboard